Dockerfile

```diff
@@ -2,7 +2,11 @@ FROM mjstealey/hs_docker_base:1.6.5
 MAINTAINER Michael J. Stealey <stealey@renci.org>
 
 ### Begin - HydroShare Development Image Additions ###
-
+RUN apt-get install -y fuse
+RUN pip install -U uwsgi

```

Comment:
Will lock down a version number when this gets ported to [hs_docker_base](https://github.com/hydroshare/hs_docker_base)